pkcs11/_pkcs11.pyx

```diff
@@ -1550,6 +1550,19 @@ cdef class lib:
         except StopIteration:
             return token
 
+    def wait_for_slot_event(self, blocking=False):
+        cdef CK_FLAGS flag = 0
+        cdef CK_SLOT_ID slot_id
+
+        if not blocking:
+            flag = 1
+
+        rv = _funclist.C_WaitForSlotEvent(flag, & slot_id, NULL)
+        assertRV(rv)
+        if rv == CKR_NO_EVENT:
+            return -1
```

Comment:
Magic values are not the right answer. This isn't C :-P You could use a sentinel value, but truthfully an exception already fulfils that need. Just let it throw the NO_EVENT exception.